build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/client_operation_documentation.rb

```diff
@@ -100,6 +101,9 @@ def option_tags(operation, api)
           if member_ref['jsonvalue']
             docstring = docstring.to_s + "<p><b>SDK automatically handles json encoding and base64 encoding for you when the required value (Hash, Array, etc.) is provided according to the description.</b></p>"
           end
+          if member_shape['document']
+            docstring = docstring.to_s + "<p>Document type which accepts Open Content - unescaped/unencoded JSON data</p>"
```

Comment:
Nit: probably better to enumerate the accepted types here and avoid the term JSON. Document types aren't limited to JSON, they just use the JSON data model (and in a protocol-agnostic way).

Reply:
I think address this